src/pudl/output/eia.py

```diff
@@ -145,20 +144,71 @@ def _out_eia__yearly_generators(
     )
 
     # If any generator data is completely empty, drop it.
-    # These are three known generators that originate from harvesting the plant and
+    # These are a handful of known generators that originate from harvesting the plant and
     # generator IDs found in the plant_id_eia_direct_support_x and
     # generator_id_direct_support_x in EIA 860 energy storage tables, in
     # order to enable foreign key relationships with these columns.
     # They do not show up in any other tables and thus lack data in all columns.
     # For more, see issue #3695 and PR #3699.
     empty_generator_ids = [
-        (9170, "3093", "2023-01-01"),
-        (34516, "SOL1", "2023-01-01"),
-        (64966, "GEN1", "2023-01-01"),
+        (1, "CFCPV"),
+        (1, "HB2PV"),
+        (1, "LUNPV"),
+        (1, "MIDPV"),
+        (1, "RDYPV"),
+        (1, "TE1PV"),
+        (9170, "3093"),
+        (34516, "SOL1"),
+        (56951, "VESTA"),
+        (57991, "PV2"),
+        (58644, "ALL"),
+        (62355, "2WPSO"),
+        (62760, "SONRI"),
+        (62975, "SYNLB"),
+        (63210, "SAINT"),
+        (69312, "EcheB"),
```

Comment:
should this guy be removed? bc its weird cased and it's CAPITALIZED friend is down below?